src/types/venues.ts

```diff
@@ -171,6 +171,7 @@ export interface BaseVenue {
   showRadio?: boolean;
   showBadges?: boolean;
   showZendesk?: boolean;
+  scheduleStartDate?: number;
```

Comment:
Why is there a new `scheduleStartDate` config item here, rather than re-using `venue.start_utc_seconds` or similar as the PR description suggests?

Reply:
now changed to Timestamp for ease of entry in firestore database

Reply:
Fair, though as asked in https://github.com/sparkletown/sparkle/pull/1419#discussion_r635709031, that doesn't really answer my question about _why_ we are introducing this new field in the first place.

In your original description for this PR you talk about using `venue.start_utc_seconds`, why not use that here?

Reply:
I think the basic reason is that @mike-lvov suggested I use `scheduleStartDate` as a field name. But since changing it to a different type than start_utc_seconds, which facilitates input by the user, it seems to make sense to keep it separate

Reply:
Copying https://github.com/sparkletown/sparkle/pull/1419#discussion_r635712102 here as well to just reply in one place:

> The answer is below (see #1419 (comment)) — I intended this to be a way to easily modify the start date directly in the firestore database. The interfact for putting in a Timestamp using the built-in calendar is much easier that searching utc_seconds. I've left it in as is, because with the type change in this revision, it seemed better not to modify venue.start_utc_seconds

Reply:
> But since changing it to a different type than `start_utc_seconds`, which facilitates input by the user, it seems to make sense to keep it separate

The 'extra overhead' of having to calculate the timestamp when hacking it into the database seems like a 'less important' tradeoff than reusing an existing field that already contains the correct data/etc.

Does the existing `start_utc_seconds` field represent the same/required date here? If so, we should be using that.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef